src/styles/base/_typography.scss

```diff
@@ -137,8 +137,10 @@ a {
     text-decoration: none;
   }
 
-  &:hover {
-    text-decoration: underline;
+  // Inline links need to be underline under WCAG 2.2 compliance
+  // More info: https://www.w3.org/WAI/WCAG22/Techniques/failures/F73
+  &:hover, &:focus, &.inline-link {
```

Comment:
I think `:focus` affects the navigator, which I think is unintended?

Reply:
Good catch! I've been thinking about what to do with the Navigator links and I believe in this case it's better to keep them underline to differentiate from group maker items.

I reviewed other elements on the UI and I prevented underline in OnThisPage, Navigator Head and Skip Nav. Also I fixed the focus style for Navigator Head. https://github.com/apple/swift-docc-render/pull/733/commits/f756425094147510469b5471580f756d7066d7ee

Can you review again? Thanks